corehq/apps/style/static/style/js/hq.helpers.js

```diff
@@ -126,6 +126,10 @@ $.fn.enableButton = function () {
            .removeAttr('disabled');
 };
 
+if (nonexistant_variable == "doubleequals") {
```

Comment:
'nonexistant_variable' is not defined. (no-undef)
Expected '===' and instead saw '=='. (eqeqeq)